### ISSUE TITLE
Fix Extension Rules lint warnings

### ISSUE THREAD

**Issue by tasneemkoushar** [PalisadoesFoundation/talawa-api#1202]
**Describe the bug**
Currently, there are too many **Extension Rules** ESLint warnings in the **develop** branch. 

Most of them are:-
"@typescript-eslint/no-empty-function",
"@typescript-eslint/no-unused-vars",

**To Reproduce**

1. Clone the **talawa-api** repository.
2. Install the project dependencies. (npm i)
3. In .eslintrc.json, change **Extension Rules**, from line 57 to 58, from "off" to "warn", just like this:-
![pic3](https://user-images.githubusercontent.com/58592306/226185951-f7bd9f2c-ae1d-4adf-9a5f-fe337ef757ea.PNG)

4. Run **npm run lint:check**

**Expected behavior**
Lint should report 0 Warnings and 0 Errors

**Actual behavior**
Lint now reports too many warnings.

**Additional details**
Multiple contributors can work together as there are too many warnings.

**Comment by tasneemkoushar** [comment]
We can merge a contributor's pull request if they are able to resolve at least 10 warnings, but we won't mark this issue as resolved until all warnings have been fixed. This allows for the creation of pull requests by many users, which we will then merge one at a time. While the remedy is minor, it is possible that two persons worked on the same warning. In that case, whoever solves it first will have their modifications merged.

**Comment by KrutikaBhatt** [comment]
Hey, can I work on this ?

**Comment by tasneemkoushar** [comment]
> Hey, can I work on this ?

Sure, I have assigned this issue to you, please go ahead.

**Comment by tasneemkoushar** [comment]
@KrutikaBhatt Please do not add `/* eslint-disable */`. I had already silenced the warnings, make PR only for the fixes in the warnings!

**Comment by KrutikaBhatt** [comment]
Yes, I have removed them and added an additional rule in `eslint.json` that will help in better development and avoid future warnings
